back-end/node.go

```diff
@@ -0,0 +1,29 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+// Package backend provides a degitx node start point
+// and contains all all back-end specific code.
+package backend
+
+import (
+	"context"
+	"log"
+
+	"cqfn.org/degitx/discovery"
+	"cqfn.org/degitx/locators"
+)
+
+// Start DeGitX node or return error
+func Start(
+	ctx context.Context,
+	node *locators.Node,
+	disc discovery.Service,
+) error {
+	log.Printf("Starting %s", node)
```

Comment:
@stepanvalyavskiy do we really need this package and function? It seems like back-end specific code and can be moved to `cmd/degitx/` (main or other file)

Reply:
the package is probably redundant but I prefer to leave Start func because it could be more complex later.